CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## To be released
 - Include pushclient module in android build
+- [iOS] set app to portrait-only
+- [iOS] implement pop-to-root
```

Comment:
Nitpick - I feel like this release note is slightly vague, perhaps needs a bit more context. Something along the lines of: pop-to-root on re-selection of tab bar item...